testnets/archwaytestnet/assetlist.json

```diff
@@ -1,20 +1,20 @@
 {
   "$schema": "../assetlist.schema.json",
-  "chain_name": "archwaytestnet",
+  "chain_name": "archway",
```

Comment:
archway is reserved for mainnet, thus it remain as 'archwaytestnet'. Not sure if it is possible to align with the chain config exactly because of this. I'd recommend updating both as necessary.